src/wms/src/main/java/org/geoserver/wms/WMSLifecycleHandler.java

```diff
@@ -18,18 +26,7 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-import org.apache.commons.io.FileUtils;
-import org.geoserver.config.GeoServerDataDirectory;
-import org.geoserver.config.impl.GeoServerLifecycleHandler;
-import org.geoserver.platform.resource.Resource;
-import org.geoserver.platform.resource.Resources;
-import org.geotools.renderer.style.FontCache;
-import org.geotools.renderer.style.ImageGraphicFactory;
-import org.geotools.renderer.style.SVGGraphicFactory;
-import org.geotools.util.logging.Logging;
-import org.springframework.context.ApplicationEvent;
-import org.springframework.context.ApplicationListener;
-import org.springframework.context.event.ContextRefreshedEvent;
```

Comment:
There seem to be a disproportionate number of import changes compared to the actual code change. Please try to limit changes to functional changes only - any reformatting should be done as a separate commit.